Example/Example for Delirium/Modules/Pie Chart/PieChartViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  PieChartViewController.swift
+//  Delirium
+//
+//  Created by Jelle Vandebeeck on 23/08/16.
+//  Copyright © 2016 CocoaPods. All rights reserved.
+//
+
+import UIKit
+import Delirium
+
+class Slice: PieChartSlice {
+    @objc var value: Float
```

Comment:
als je class gebruikt dan is die @objc hier niet nodig. Dus vind ik class wel mooier dan?